src/models/funds

```diff
@@ -0,0 +1,88 @@
+const { sum } = require('lodash');
+
+const mongoose = require('mongoose');
+
+const fundSchema = mongoose.Schema(
+  {
+    puplisherId: {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'BaseUser',
+    },
+    title: {
+      type: String,
+      required: true,
+    },
+    content: {
+      type: String,
+      required: true,
+    },
+    category: {
+      type: String,
+      enum: [
+        'No Poverty',
```

Comment:
same here this is the same enum we have in events so lets move it to a shared variable folder and use it here, please contact @mehmettbaki and agree on what the naming will be for this file/folder that u will save the variables in